src/content/community/support/index.md

```diff
@@ -107,4 +107,8 @@ Transactions on Ethereum can sometimes get stuck if you have submitted a lower t
 
 #### How do I mine Ethereum? {#mining-ethereum}
 
-We do not recommend buying mining equipment if you are not already mining Ethereum. In ~Q3/Q4 2022, [The Merge](/upgrades/merge/) will happen, switching Ethereum from proof-of-work to proof-of-stake. This change means mining Ethereum will no longer be possible. If you are still interested in mining Ethereum, you can seek help from mining communities, such as [/r/EtherMining](https://www.reddit.com/r/EtherMining/).
+Ethereum mining is no longer possible. Ethereum mining was switched off when the network moved from proof-of-work to proof-of-stake. Now, instead of miners, Ethereum has validators. Validators stake ether and receive staking rewards for securing the network.
```

Comment:
```suggestion
Ethereum mining is no longer possible. Mining was switched off when Ethereum moved from proof-of-work to proof-of-stake. Now, instead of miners, Ethereum has validators. Validators stake ETH and receive staking rewards for securing the network.
```